gitensite/apps/bookinfo/management/commands/load_repos.py

```diff
@@ -24,10 +24,35 @@ def handle(self, rdf_library, start=0, *args, **kwargs):
                 continue
             try:
                 (book,created) = Book.objects.get_or_create(book_id=int(pg_id), repo_name=repo_name)
+
                 metadata=BookMetadata(book,rdf_library=rdf_library, enrich=should_enrich)
+
```

Comment:
it seems like the following code should duplicate the code that handles a post. so maybe better to put it on the model so both management command and post handler can call book.load_metadata(metadata)